prober/http.go

```diff
@@ -342,6 +340,16 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 		return false
 	}
 
+	// Do not move the following variable to global scope. The cases.Caser returned by
+	// calling cases.Title *cannot* be shared among goroutines. This might happen when
+	// Prometheus tries to scrape multiple targets at the same time. From the docs:
+	//
+	// A Caser may be stateful and should therefore not be shared between goroutines.
+	//
+	// Issue: https://github.com/prometheus/blackbox_exporter/issues/922
```

Comment:
I don't think it is necessary to point to the gh issue here

Reply:
I felt it was necessary to keep the reference close to the code because the original version is what I would have written, too. The wording in the docs, "a Caser may", sounds like if that were the actual case, the individual Caser implementations would spell that out. Given that in this particular instance there no such note, _I_ would have assumed that this Caser is not stateful.

Thanks for the review and merging this!